README.md

```diff
@@ -14,18 +14,20 @@ swift ./validate.swift
 
 When validation succeeds, commit your changes and submit your pull request! Your package(s) will appear in the index within a few minutes.
 
+> **Note:** If submitting your own packages, don't forget to add shields.io badges to your package's README to always have up to date swift version and platform compatibility information readily available. Once your package appears in the index, use the "Do you maintain this package?" link in the right-hand sidebar of your package page and use the provided markdown.
+
```

Comment:
I think we could even link to a badge image here?

Reply:
It was hard to get it looking good with GitHub's markdown formatting. I'm doing more on the Add a Package page.

Reply:
was thinking like this perhaps?

<img width="765" alt="CleanShot 2022-01-18 at 15 47 05@2x" src="https://user-images.githubusercontent.com/65520/149959608-008086ac-969f-4bad-b3d0-7e72bfd17c22.png">

Reply:
Actually hold on. I was trying without the Blockquote, let me try with it.